apps/team-awesome-web/src/components/Form/team-member-form/TeamMemberForm.tsx

```diff
@@ -4,10 +4,9 @@ import { Button } from "../../button";
 import { Input } from "../input/Input";
 import * as Styled from "./TeamMemberForm.styled";
 import { StoreContext } from "../../store-context";
-import { TeamMember } from "../../../types";
 
 export const defaultMember = {
-  id: 0,
+  id: "0",
```

Comment:
Just curious, why did you change this `id` to a string?

Reply:
I changed it to string because in graphQL GET_TIME_ENTRIES i also made it a string yesterday with Fabiano and Herman. When i pushed it this morning a type error came up saying it cannot compare string as a number or something when using the default entry because i re-typed it as a string. Could be number either way.